rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1107,11 +1105,6 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		}
 	});
 
-	if (const int lib_mode_button_id = lib_mode_bg->checkedId(); lib_mode_button_id >= 0)
-	{
-		on_lib_button_clicked(lib_mode_button_id);
```

Comment:
this needs to be called anyway, since we want to presort the libs

Reply:
1 - We dont need to sort them anymore because they are sorted in g_prx_list.
2 - Calling it invalidates selection state of the checkboxes which we only want after switching mode.

Reply:
alright